### PR TITLE
#134 Adding test to validate the correctness of eth_estimateGas funct…

### PR DESCRIPTION
**Pull request by ggarri**
Adding a test unit which validates the correctness of eth_estimateGas method

### CODE REVIEW

File: truffle/test/05_eth_api.js [hunk 1]
```diff
@@ -15,4 +33,41 @@ describe('Ethereum API', () => {
...
+          arguments: []
+        }).send({ gas: estimatedGas })
+            .on('error', (err) => {
+              reject(err)
+            })
+            .on('confirmation', (confirmationNumber, receipt) => {
+              resolve(receipt);
+            })
+      });
+    };
```
Comment (EnchanterIO) [inline truffle/test/05_eth_api.js]: can't we use ES6 syntax here?

File: truffle/test/05_eth_api.js [hunk 2]
```diff
@@ -2,9 +2,27 @@
...
+
 describe('Ethereum API', () => {
+  let _web3;
+  let _debug;
+  let ROOT_ACCOUNT = extractEnvAccountAndPwd(process.env.NETWORK).from;
+
+  it("should initialize library web3", async () => {
+    _web3 = new Web3(web3._provider.host, null, {
+      defaultAccount: ROOT_ACCOUNT,
+      defaultGasPrice: "500000000000"
```
Comment (EnchanterIO) [inline truffle/test/05_eth_api.js]: can we add this to the .env file? And read it from all the tests or usages? This value seems to be repeated quiet a lot and we may change it soon. Will be easier to refactor.

File: truffle/test/05_eth_api.js (hunk 1 above)
Comment (ggarri) [reply truffle/test/05_eth_api.js]: In this case, it is not possible unfortunally

File: truffle/test/05_eth_api.js (hunk 2 above)
Comment (ggarri) [reply truffle/test/05_eth_api.js]: It is not the `.env`, it is the truffle.config and I tried and it is not possible unless I add logic on the migrations to do that and it seems annoying to do so

Comment (EnchanterIO) [reply truffle/test/05_eth_api.js]: if it's in the truffle config then u don't have to define it here btw. but is a little thing, will be easy to change one way or the other. approved

File: truffle/test/05_eth_api.js (hunk 1 above)
Comment (EnchanterIO) [reply truffle/test/05_eth_api.js]: promises ftw

**Pull request merged**